feature/google/src/bc5SupportDev/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -60,7 +61,9 @@ private const val RECONNECT_TIMER_MAX_TIME_MILLISECONDS = 1000L * 60L * 15L // 1
 class BillingWrapper(
     private val clientFactory: ClientFactory,
     private val mainHandler: Handler,
-    private val deviceCache: DeviceCache
+    private val deviceCache: DeviceCache,
+    @Suppress("unused")
+    private val diagnosticsTracker: DiagnosticsTracker?
```

Comment:
I foresee a complicated merge conflict with BC5 once we support it... So I decided to just add the functionality to the BC4 version for now. My plan is to create a ticket to just do this again in the BC5 integration branch after it's been merged to main and to that branch. Lmk if you have any concerns over that.